### ISSUE TITLE
.puml and .svg files are generated in the wrong output dir in android projects

### ISSUE THREAD

**Issue by jprogramer** [talsma-ict/umldoclet#160]
## Expected Behavior
The files generated by the standard doclet and umldoclet should goes to the javadoc destination directory, for example:
In a project with a layout like this:
```
DocletTest
    |____app
        |____src
            |____androidTest
            |____main
                |____java
                    |____org.comp.product
                |____AndroidManifest.xml
            |____test
        |____build.gradle // App build script configuration file
    |____build.gradle // Top-level build script configuration file
    |____settings.gradle
``` 
all the javadoc files should goes to DocletTest/app/output
## Actual Behavior
all files are generated into DocletTest/app/**build/docs/javadoc**/org/comp/product,  
except .puml and .svg files which are generated to DocletTest/app/org/comp/product

## Steps to Reproduce the Problem

  1. create a new android project in Android Studio
  2. append this configuration to the App build script configuration file:
```
configurations {
    umlDoclet
}

dependencies {
    umlDoclet "nl.talsmasoftware:umldoclet:1.1.3"
}

task myDocs(type: Javadoc) {
    source = android.sourceSets.main.java.sourceFiles
    options.docletpath = configurations.umlDoclet.files.asType(List)
    println(configurations.umlDoclet.files.asType(List))
    options.doclet = "nl.talsmasoftware.umldoclet.UMLDoclet"
}
```
  3. run Gradle task 'myDocs' on the app sub-project

## Specifications
Gradle Version 5.4
Android Studio 3.4
    Build #AI-183.5429.30.34.5452501, built on April 10, 2019
    JRE: 1.8.0_152-release-1343-b01 amd64
    JVM: OpenJDK 64-Bit Server VM by JetBrains s.r.o
    Windows 10 10.0

**Comment by sjoerdtalsma** [comment]
Thank you for bringing this to our attention!
Could you please attach the log from the gradle debug output when the javadoc is run?

**Comment by sjoerdtalsma** [comment]
Fixed in `1.1.4` which should become available in maven within the next couple of hours.
Thanks again for reporting this 👍

**Comment by jprogramer** [comment]
Thank you so much for fixing this so quickly!
and sorry for not attaching the log earlier :(

Keep up the great work guys! 👍
